### ISSUE TITLE
Incompatible with Microsoft.Bot.Builder v3.14.1.1

### ISSUE THREAD

**Issue by GeertHuls** [ObjectivityLtd/Bot.Tests.Stories.Xunit#2]
Hi,

Currently the xunit project depends on Microsoft.Bot.Builder v3.12.2.4 and upgrading to Microsoft.Bot.Builder v3.14.1.1 is not possible due to breaking changes.

In particular, the StateClient mock setup in MockConnectorFactory doesn't compile anymore.
Any advice how to fix this?

Regards.

**Comment by jkolonko** [comment]
Hi,

I just released a new version 0.3.1 that should be compatible with current stable bot framework release.  I think the commit describes how I hopefully managed to fix the issue. It was caused by more than that _MockConnectorFactory_ changes - also _LuisDialog_ now has some _RemoveSensitiveData_ mechanism that required some more mocking internally.

It took a while to fix that because the workaround was straightforward, I did not rely on any of the new features so I could live with earlier bot framework versions. 

Regards,
Jacek
